### ISSUE TITLE
"sbt assembly" fails due to dependency problem

### ISSUE THREAD

**Issue by nobusue** [ldaniels528/trifecta#17]
at 1fc3462d8f0b260a40d3e31dd3a4145accb277e8

I've faced build problem as follows:

    [error] (*:update) sbt.ResolveException: unresolved dependency: com.ldaniels528#commons-helpers_2.11;0.1.0: not found
    [error] unresolved dependency: com.ldaniels528#tabular_2.11;0.1.1: not found
    [error] unresolved dependency: clj-time#clj-time;0.4.1: not found
    [error] unresolved dependency: compojure#compojure;1.1.3: not found
    [error] unresolved dependency: clout#clout;1.0.1: not found
    [error] unresolved dependency: ring#ring-core;1.1.5: not found
    [error] unresolved dependency: hiccup#hiccup;0.3.6: not found
    [error] unresolved dependency: ring#ring-devel;0.3.11: not found
    [error] unresolved dependency: clj-stacktrace#clj-stacktrace;0.2.2: not found
    [error] unresolved dependency: ring#ring-jetty-adapter;0.3.11: not found
    [error] unresolved dependency: ring#ring-servlet;0.3.11: not found
    [error] unresolved dependency: com.twitter#carbonite;1.4.0: not found

I've found to solve part of these by recovering

    resolvers += "Clojure Releases" at "http://build.clojure.org/releases/"

But, I couldn't find two artifacts: 

    "com.ldaniels528" %% "commons-helpers" % "0.1.0",
    "com.ldaniels528" %% "tabular" % "0.1.1"

Where can I get them?

**Comment by ldaniels528** [comment]
These are projects that are also located on my Github account.

**Comment by ldaniels528** [comment]
https://github.com/ldaniels528/commons-helpers
https://github.com/ldaniels528/tabular

**Comment by nobusue** [comment]
Thank you.
I got them and could build successfully, but it was not straightforward...

(1) They aren't available as maven repository
I've found repo for "tabular", but "0.1.1" doesn't exist.
https://github.com/ldaniels528/tabular/tree/gh-pages/repo/com.ldaniels528
Also there is no repo for "commons-helpers".

So, I cloned them and install local ivy2 cache via "sbt publish-local"

(2) Incompatible API
Both "tabular" and "commons-helpers" have been changed package name, so I couldn't build trifecta with HEAD commit.
I guess compatible commits are:
https://github.com/ldaniels528/tabular/releases/tag/v0.1.1
https://github.com/ldaniels528/commons-helpers/commit/93509eb84b650a4c78f9cb925b0db2e47c659927

Is that right?
